src/net_dev.rs

```diff
@@ -0,0 +1,148 @@
+//! Net device.
```

Comment:
Shall we add two functions to start and stop a specific device?

Reply:
Yes.